### PR TITLE
remove RHEL base image support, fix missing Jolokia Version

### PR DESCRIPTION
**Pull request by agebhar1**
* remove support for RHEL java image
* fix Jolokia version on Karaf image(s)
* refresh/-build »Dockerfile«s

### CODE REVIEW

File: karaf/images.yml [hunk 1]
```diff
@@ -18,16 +18,6 @@ config:
       description: "CentOS S2I Karaf builder image with OpenJDK 8"
       version:
         maven: "3.6.3"
-        jolokia: "1.6.3"
+        jolokia: "1.6.2"
```
Comment (rhuss) [inline karaf/images.yml:21]: why downgrade ?

File: karaf/images/centos/run-java.sh [hunk 2]
```diff
@@ -33,7 +33,7 @@
 #                             https://www.youtube.com/watch?v=w1rZOY5gbvk
 #                             https://vimeo.com/album/4133413/video/181900266
 # Also note that heap is only a small portion of the memory used by a JVM. There are lot
-# of other memory areas (metadata, thread, code cache, ...) which addes to the overall
+# of other memory areas (metadata, thread, code cache, ...) which adds to the overall
```
Comment (rhuss) [inline karaf/images/centos/run-java.sh:36]: How was this file updated ? Normally you should update karaf/fish-pepper.yml, too with the proper version for run-java and then re-run fish-pepper.

File: karaf/images.yml (hunk 1 above)
Comment (agebhar1) [reply karaf/images.yml:21]: grafik
Version1.6.3 does not exists (yet).

Comment (agebhar1) [reply karaf/images.yml:21]: The Java part refers to 1.6.2.

File: karaf/images/centos/run-java.sh (hunk 2 above)
Comment (agebhar1) [reply karaf/images/centos/run-java.sh:36]: I run fish-pepper in _karaf_ directory. In 526cdb3a07b0767be2681792d587c7af4a800dea versions are updated in the karaf directory but I assume fish-pepper didn't run. The files in the _java_ directory has been updated.

File: karaf/images.yml (hunk 1 above)
Comment (rhuss) [reply karaf/images.yml:21]: sorry, you are absolutely right, 1.6.2 is the latest (i should know it 😬 )

File: karaf/images/centos/run-java.sh (hunk 2 above)
Comment (rhuss) [reply karaf/images/centos/run-java.sh:36]: 👍

**Comment by agebhar1** [comment]
You're welcome!

**Pull request merged**